### DISCUSSION TITLE
Weird-to-me Sabayon Notification

### DISCUSSION THREAD

**Discussion by VolvoBirkenstock** [mocaccinoOS/mocaccino#86] [General] [answered]
Hi, Sabayon/Mocaccino Gurus.  

I know this is a very weird question, but I just received a very strange and unexpected Sabayon notification on a system that is still running Sabayon.  For the first time in an extremely long time, I am being informed that "there are 1067 updates available."  Is this for real?  I'm not going to do anything until I hear back from you, but I have been operating under the impression that Sabayon was no longer supported.  If Sabayon is no longer supported, then why am I receiving this notification? 

Thank you very much in advance.

**Comment by joostruis** [comment]
No idea. We did not touch anything regarding Entropy / Sabayon.

**Reply by VolvoBirkenstock** [reply]
Thanks, Joost.  I will ignore the alleged updates and do nothing.

**Reply by joostruis** [reply]
What also could be the case is that some mirrors are now gone and some very outdated mirror has an old Entropy DB and actually causes packages to downgrade. (in theory)

**Reply by VolvoBirkenstock** [reply]
I am very curious about this.  The paranoid side of my brain is wondering if someone has done this malevolently.  Is there an "equo" command line with arguments that will display all the update names and versions?  If so, I can run the equo command and paste the results here.  I can see the names and versions in Rigo, but cannot easily cut-and-paste that output.  I suppose I could perform a screen grab, but would prefer to provide text instead of many images. 

"equo query updates --verbose" returns:
╠  @@ Available Updates
╠   Packages to update:  1067
╠   Packages to remove:  66
╠   Packages already up-to-date:  1082
╠   Critical updates found:       False

"equo query list" and "equo query updates list" both return the "help" output, but indicate that "list" is an acceptable argument. Perhaps my syntax is incorrect. I stumbled on "equo query list available sabayonlinux.org --verbose | wc -l" which returned 14,187 entries. 

The very first entry in Rigo when viewing what occurs after clicking the "more info" button is "A52dec **0.7.4-r7~1**" followed by the statement "library for decoding ATSC A/52 streams used in DVD March 08, 2020, from sabayonlinux.org **0.7.4-r8** is installed".  So this does look like a downgrade.  

I will drop this if you believe my paranoia is not worth our time. ;-) 
Again, thank you very much, Joost.

**Reply by joostruis** [reply]
If I recall correctly, equo upgrade -p (p for pretend) should perhaps reveal a bit more.

**Reply by VolvoBirkenstock** [reply]
Joost, you assessed the situation perfectly.  I performed "equo upgrade -p" and routed stdout to a file.  I went through the file, picking out five entries at random, and found that each of those entries was a downgrade from a previously installed revision.  

I have disabled the notification applet on that Sabayon system.  

Thank you very much!

**Comment by joostruis** [comment, marked as answer]
So the conclusion here is: More and more mirrors shutdown and as a result you fallback on a mirror with an outdated Entropy database that actually triggers a downgrade of packages.
